app/src/main/java/com/example/dndwiki/MainActivity.kt

```diff
@@ -16,6 +21,11 @@ class MainActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        startKoin {
```

Comment:
Put the initialisation of Koin in your Application class (you still need to create one) as per documentation

Don't forget to add your application class to manifest.xml